src/sas/qtgui/Utilities/Reports/UI/ReportDialogUI.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Report: Fitting</string>
+   <string>Report</string>

```

Comment:
Suggestion: set the window title from the `ReportDialog` class based on which perspective is shown.

Reply:
The reports do have titles in their HTML, so not sure it matters that much, however, could easily do along with the filename stuff.

Reply:
Good point. Disregard.